flycheck-pos-tip.el

```diff
@@ -65,8 +66,31 @@ The function should be a no-op in this case."
   :type 'function
   :package-version '(flycheck-pos-tip . "0.2"))
 
+(defcustom flycheck-pos-tip-tty-show-function
+  #'flycheck-pos-tip-tty-show
```

Comment:
Shouldn't we just use `flycheck-display-error-messages` as default?  After all, we already have this feature in Flycheck itself, so let's use it :)